data/json/npcs/NPC_Brigitte_LaCroix.json

```diff
@@ -0,0 +1,343 @@
+      "id": "NC_BONE_SEER",
+      "name": "Bone Seer",
+      "job_description": "I wish to sing the song of the Bones",
+      "traits": [
+        { "trait": "OUTDOORSMAN" },
+        { "trait": "HEAVYSLEEPER" },
+        { "trait": "GOODCARDIO" },
+        { "trait": "LIGHTSTEP" },
+        { "trait": "ROBUST" },
+		{ "trait": "seer_mark" },
+        { "group": "Appearance_African" },
+		{ "trait": "hair_black_mohawk" }
+      ],
+      "common": false,
+      "bonus_per": { "rng": [ 0, 2 ] },
+      "bonus_str": { "rng": [ 0, 2 ] },
+      "bonus_int": { "rng": [ 0, 2 ] },
+      "worn_override": "BONE_SEER_Brigitte_worn",
+      "weapon_override": "BONE_SEER_Brigitte_wield",
+      "shopkeeper_item_group": "NC_BONE_SEER_misc",
+      "skills": [
+        {
+          "skill": "ALL",
+          "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": -2 }, { "one_in": 4 } ] } ] }
+        },
+        { "skill": "survival", "bonus": { "rng": [ 2, 4 ] } },
+        { "skill": "dodge", "bonus": { "rng": [ 4, 8 ] } },
+        { "skill": "melee", "bonus": { "rng": [ 4, 6 ] } },
+        { "skill": "bashing", "bonus": { "rng": [ 3, 6 ] } },
+        { "skill": "cutting", "bonus": { "rng": [ 3, 6 ] } }
+      ]
+    },
+    {
+      "type": "item_group",
+      "id": "BONE_SEER_Brigitte_worn",
+      "subtype": "collection",
+      "entries": [
+        { "item": "loincloth_leather" },
+        { "item": "socks" },
+        { "item": "cloak" },
+        { "item": "armor_blarmor" },
+        { "item": "balclava" },
+        { "item": "gloves_liner" },
+        { "item": "helmet_bone" },
+        { "item": "gauntlets_bone" },
+        { "item": "boots_bone" },
+        { "item": "armguard_bone" },
+        { "item": "survivor_goggles" },
+        { "item": "survivor_pack" }
+      ]
+    },
+    {
+      "type": "item_group",
+      "id": "BONE_SEER_Brigitte_wield",
+      "subtype": "collection",
+      "entries": [ { "item": "2h_flail_steel" } ]
+    },
+    {
+      "type": "item_group",
+      "id": "NC_BONE_SEER_misc",
+      "items": [
+        [ "bone_plate", 10 ],
+        [ "needle_bone", 40 ],
+        [ "bone_glue", 20 ],
+        [ "bone_flute", 10 ],
+        [ "bone_human", 10 ],
+        [ "broth_bone", 60 ],
+        [ "water_clean", 90 ]
+      ]
+    },
+	{
+    "type": "mutation",
+    "id": "seer_mark",
+    "name": "Mark of the Seer",
+    "points": 1,
+    "description": "A strange occult symbol carved into your hand by Brigitte LaCroix",
+    "starting_trait": false,
+	"valid": false,
+	"purifiable": false
+    },
+    {
+      "type": "npc",
+      "id": "SEER_Brigitte_LaCroix",
+      "//": "A mysterious member of a doomsday cult.",
+      "name_unique": "Brigitte LaCroix",
+      "gender": "female",
+      "name_suffix": "Bone Seer",
+      "class": "NC_BONE_SEER",
+      "attitude": 7,
+      "mission": 3,
+      "chat": "TALK_BONE_SEER",
+      "faction": "wasteland_scavengers",
```

Comment:
We're discouraging the use of "wasteland_scavengers" as a catch-all faction for NPCs.  She could either have no faction, or her own faction with her being the only member.  The first option is easier.

Reply:
When I left it open it gave me ann error